pkg/kubelet/logs/container_log_manager.go

```diff
@@ -412,8 +412,9 @@ func (c *containerLogManager) rotateLatestLog(id, log string) error {
 			// This shouldn't happen.
 			// Report an error if this happens, because we will lose original
 			// log.
-			klog.Errorf("Failed to rename rotated log %q back to %q: %v, reopen container log error: %v", rotated, log, renameErr, err)
+			klog.ErrorS(renameErr, "Failed to rename rotated log %q back to %q: %v", "rotatedLog", rotated, "log", log)
 		}
+		klog.ErrorS(err, "Failed to reopen container log", "containerID", id)
```

Comment:
```suggestion
```

This line is not necessary, the error below is being returned as opposed to logged.